common/services/prismic/parsers.js

```diff
@@ -466,15 +467,13 @@ export function parseFormat(frag: Object): ?Format {
     : null;
 }
 
-function parseLink(url): ?string {
-  if (url) {
-    if (url.link_type === 'Web' || url.link_type === 'Media') {
-      return url.url;
-    } else if (url.link_type === 'Document' && isDocumentLink(url)) {
-      return `/${url.type}/${url.id}`;
+export function parseLink(link?: PrismicLink): ?string {
+  if (link) {
+    if (link.link_type === 'Web' || link.link_type === 'Media') {
+      return link.url;
+    } else if (link.link_type === 'Document' && isDocumentLink(link)) {
+      return `/${link.type}/${link.id}`;
```

Comment:
`link.type` is not always correct. We should really being using the [`linkResolver`](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/services/prismic/link-resolver.js) here.